frontend/rpc.bal

```diff
@@ -86,106 +86,92 @@ final stub:CheckoutServiceClient checkoutClient = check new (string `http://${ch
 isolated function getSupportedCurrencies() returns string[]|grpc:Error {
     stub:GetSupportedCurrenciesResponse|grpc:Error supportedCurrencies = currencyClient->GetSupportedCurrencies({});
     if supportedCurrencies is grpc:Error {
-        log:printError("failed to call getSupportedCurrencies from currency service", 'error = supportedCurrencies);
+        log:printError("failed to get supported currencies from currency service", supportedCurrencies);
```

Comment:
```suggestion
        log:printError("failed to get supported currencies from the currency service", supportedCurrencies);
```

May be applicable in other places too.

Reply:
Fixed in https://github.com/ballerina-guides/gcp-microservices-demo/pull/65/commits/e746edee6815377481f762a558e106b9e16d715b